apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -126,6 +128,7 @@ export default function manageStudents(state=initialState, action) {
       editingData: {
         ...state.editingData,
         [action.studentId]: {
+          ...state.editingData[action.studentId],
```

Comment:
It looks like you are updating the redux state with the data that's currently in the row whenever an input change occurs, rather than waiting until the data is sent to the server to save it. This seems right.

Reply:
This is actually just the initial editing state (not the saved student data) which is initially a copy of the saved data. No need to wait for server to start editing, just to actually save.